solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/TestLTRScoringModel.java

```diff
@@ -72,20 +71,14 @@ public void getInstanceTest() {
 
   @Test
   public void nullFeatureWeightsTest() {
-    final ModelException expectedException = 
-        new ModelException("Model test2 doesn't contain any weights");
-    try {
-      final List<Feature> features = getFeatures(new String[] 
-          {"constant1", "constant5"});
-      final List<Normalizer> norms = 
+    final List<Feature> features = getFeatures(new String[] 
+        {"constant1", "constant5"});
+    final List<Normalizer> norms = 
         new ArrayList<Normalizer>(
             Collections.nCopies(features.size(),IdentityNormalizer.INSTANCE));
-      final LTRScoringModel meta = new RankSVMModel("test2",
-          features, norms, "test", fstore.getFeatures(), null);
-      fail("unexpectedly got here instead of catching "+expectedException);
-    } catch (ModelException actualException) {
-      assertEquals(expectedException.toString(), actualException.toString());
-    }
+    final LTRScoringModel meta = new RankSVMModel("test2",
```

Comment:
This unit test and missingFeatureWeightTest have been changed to check if the objects are made instead of checking for an error. Shouldn't we move the error generating code into validate though and keep the tests the same?